src/components/Comments.jsx

```diff
@@ -44,6 +50,35 @@ export const Comments = ({ singleReview }) => {
     }
   };
 
+  const handleDeleteComment = (comment_id) => {
+    getUsers()
+      .then((users) => {
+        console.log(users);
+        return users;
+      })
+      .then((users) => {
+        if (users.username !== comments.author) {
+          alert("Cannot delete another users comment");
```

Comment:
This alert didn't pop up for me and I could still click delete, you need to completely stop a user from even being able to start trying to do that

Reply:
How do I even set a user though if we haven't set up any login user authentication?